### ISSUE TITLE
synopsis incorrectly recommends qw(schema resultset rset)

### ISSUE THREAD

**Issue by jasonblewis** [ironcamel/Dancer2-Plugin-DBIC#17]
using Dancer2::Plugin::DBIC with Dancer2 0.20000 as suggested in the synopsis, you will give errors like:

```
Error while loading app.psgi: Can't call method "keywords" on an undefined value at ~/perl5/perlbrew/perls/perl-5.22.0/lib/site_perl/5.22.0/Dancer2/Plugin.pm line 501.
```

The correct way to use the plugin with d2 is now:
```
use Dancer2::Plugin::DBIC;
```

**Comment by ironcamel** [comment]
Thanks for the bug report @jasonblewis

**Comment by ironcamel** [comment]
I fixed this and released it in v0.0014 https://metacpan.org/release/IRONCAMEL/Dancer2-Plugin-DBIC-0.0014
